src/app/clusters/access-control-server/access-control-server.cpp

```diff
@@ -476,7 +483,28 @@ CHIP_ERROR AccessControlAttribute::ReadAcl(AttributeValueEncoder & aEncoder)
 
 CHIP_ERROR AccessControlAttribute::ReadExtension(AttributeValueEncoder & aEncoder)
 {
-    return aEncoder.EncodeEmptyList();
+    auto & storage = Server::GetInstance().GetPersistentStorage();
+    DefaultStorageKeyAllocator key;
+
+    auto & fabrics = Server::GetInstance().GetFabricTable();
+
+    return aEncoder.EncodeList([&](const auto & encoder) -> CHIP_ERROR {
+        for (auto it = fabrics.begin(); it != fabrics.end(); ++it)
```

Comment:
```suggestion
        for (auto & fabric : fabrics)
```

And then `fabric.GetFabricIndex()` etc....

Reply:
Oh if that works, sure, I'll do it.

Reply:
Done.